### DISCUSSION TITLE
Weird output from Jest + Turbo since a recent update (Feb 2024)

### DISCUSSION THREAD

**Discussion by jcollum-nutrien** [vercel/turborepo#7377] [Help] [answered]
The jest command is causing a weird space in the turbo output. Setting concurrency to 1 doesn't fix it, but the output is much worse without concurrency=1. 

Note the orange arrow. Any way to fix this? 

&nbsp; 
**normal test output:** 
![iScreen Shoter - iTerm2 - 240213150831](https://github.com/vercel/turbo/assets/99687439/eb537e96-f7da-4821-9109-a07c6c62be17)

&nbsp;
&nbsp;
**weird gap in turbo output with concurrency=1** 

![iScreen Shoter - iTerm2 - 240213150657](https://github.com/vercel/turbo/assets/99687439/6818aae2-6cfd-450e-95d8-7eb5da1d2cc9)

&nbsp;
&nbsp;
**even worse with concurrency unset:** 

![iScreen Shoter - iTerm2 - 240213151452](https://github.com/vercel/turbo/assets/99687439/87110d14-15bb-4f05-9a97-ece3ba2f6f15)

**Comment by anthonyshew** [comment, marked as answer]
Jest (and some other tooling) try to wholesale overwrite the terminal at times while it is writing its output. We've released a new terminal UI in 2.0 that solves for most (all?) of this problem. Would love for you to give it a try and provide feedback in Issues if you could!
